### PR TITLE
Enable tests for the preopens dir with rootfs and mounts

### PR DESCRIPTION
**Pull request by hydai**

### CODE REVIEW

**Comment by hydai** [comment]
The CI is green now, @CaptainVincent PTAL

File: demo/README.md [hunk 1]
```diff
@@ -1,11 +1,12 @@
 # Demo
 
 ## Index
-- [Hyper Cleint / Server](https://github.com/CaptainVincent/runwasi/tree/main/demo#demo-1-hyper-cleint--server)
-- [Reqwest](https://github.com/CaptainVincent/runwasi/tree/main/demo#demo-2-reqwest)
-- [Database](https://github.com/CaptainVincent/runwasi/tree/main/demo#case-3-database)
-- [Microservice with Database](https://github.com/CaptainVincent/runwasi/tree/main/demo#case-4-microservice-with-database)
-- [WASI NN](https://github.com/CaptainVincent/runwasi/tree/main/demo#case-5-wasi-nn-x86-only)
+- [Hyper Cleint / Server](https://github.com/second-state/runwasi/tree/main/demo#demo-1-hyper-cleint--server)
```
Comment (CaptainVincent) [inline demo/README.md:4]: I think the link should be #title only. No need full url.

**Review by CaptainVincent — APPROVED** [review]
LGTM.

**Review by rumpl — APPROVED** [review]
🎉

**Pull request merged**